src/LinuxTracing/LibunwindstackUnwinderTest.cpp

```diff
@@ -0,0 +1,211 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <absl/strings/str_format.h>
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include "LibunwindstackMaps.h"
+#include "LibunwindstackUnwinder.h"
+#include "Test/Path.h"
+
+namespace {}  // namespace
+
+namespace orbit_linux_tracing {
+
+std::unique_ptr<LibunwindstackMaps> CreateFakeMapsEntry(std::string_view target) {
+  std::string maps_file_entry = absl::StrFormat(
+      "000000000000-000000001000 r--p 00000000 fe:00 123 %1$s\n"
+      "000000001000-000000003000 r-xp 00001000 fe:00 123 %1$s\n"
+      "000000003000-000000004000 r--p 00003000 fe:00 123 %1$s\n"
+      "000000004000-000000005000 r--p 00003000 fe:00 123 %1$s\n"
+      "000000005000-000000006000 rw-p 00004000 fe:00 123 %1$s",
+      (orbit_test::GetTestdataDir() / target).string());
+  return LibunwindstackMaps::ParseMaps(maps_file_entry);
+}
+
+// 0000000000001215 <_Z9every_1usv>:
+//    1215:       48 c7 44 24 f8 00 00    movq   $0x0,-0x8(%rsp)
+//    121c:       00 00
```

Comment:
These zero are part of the previous instruction. I guess this is from the output of `objdump` but can we reformat?

Reply:
Same in multiple places below.

Reply:
Done.